src/python/pants/engine/internals/native_engine.pyi

```diff
@@ -79,25 +173,24 @@ class PyGeneratorResponseGetMulti:
     def __init__(self, **kwargs: Any) -> None: ...
 
 class PyNailgunServer:
-    def __init__(self, **kwargs: Any) -> None: ...
+    pass
```

Comment:
This doesn't implement `__new__` in `interface.rs`, so this type hint was a lie.